base/show.jl

```diff
@@ -1131,6 +1131,26 @@ const expr_parens = Dict(:tuple=>('(',')'), :vcat=>('[',']'),
 
 is_id_start_char(c::AbstractChar) = ccall(:jl_id_start_char, Cint, (UInt32,), c) != 0
 is_id_char(c::AbstractChar) = ccall(:jl_id_char, Cint, (UInt32,), c) != 0
+
+"""
+     isidentifier(s) -> Bool
+
+Returns whether the symbol or string `s` contains characters that are parsed as
+a valid identifier in Julia code.
+
+Internally Julia allows any sequence of characters in a `Symbol` (except `\0`s),
+and macros automatically use variable names containing `#` in order to avoid
+naming collision with the surrounding code. In order for the parser to
+recognize a variable, it uses a limited set of characters (greatly extended by
+Unicode). `isidentifier()` makes it possible to query the parser directly
+whether a symbol contains valid characters.
+
+# Examples
+```jldoctest
+julia> Meta.isidentifier(:x), Meta.isidentifier("1x")
```

Comment:
```suggestion
julia> isidentifier(:x), isidentifier("1x")
```

Reply:
They won't be available at the REPL natively

Reply:
I thought it would, as you seem to export it from Base.

Reply:
ah, whoops, I forgot to drop that file from the commit